Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/Users/UsersController.cs

```diff
@@ -521,7 +521,9 @@ private bool CanUpdateUsername(UserInfo user)
 
             var parsedSearchText = string.IsNullOrEmpty(usersContract.SearchText) ? "" : SearchTextFilter.CleanWildcards(usersContract.SearchText.Trim());
 
-            usersContract.SearchText = string.Format("{0}*", parsedSearchText);
+            usersContract.SearchText = DateTime.TryParse(parsedSearchText, out var _)
```

Comment:
This is now doing additional attempted date parsing and object creation, If we support this we should avoid multiple DateTime.TryParse calls over the same content

Reply:
Agreed @mitchelsellers

Reply:
the main issue IMO: it is not supporting localized date formats